src/screens/FeedScreen/ContentPoll.js

```diff
@@ -334,7 +333,7 @@ const stylesComponent = (pollLength) =>
     },
     voteFont: {
       fontSize: normalizeFontSize(12),
-      color: '#828282',
+      color: COLORS.blackgrey,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color value has been changed from a hard-coded value to a constant. This is a good practice as it improves maintainability and consistency across the application.
```diff
-      color: '#828282',
+      color: COLORS.blackgrey,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->